Userland/Services/SpiceAgent/main.cpp

```diff
@@ -18,6 +18,7 @@ ErrorOr<int> serenity_main(Main::Arguments)
 
     TRY(Core::System::pledge("unix rpath wpath stdio sendfd recvfd"));
     TRY(Core::System::unveil(SPICE_DEVICE, "rw"sv));
+    TRY(Core::System::unveil("/proc/all", "rw"));
```

Comment:
I don't think we need `rw` access to `/proc/all` here, `r` should be enough :)

Reply:
Right, I'll fix this.